FastSimulation/Tracking/python/GlobalPixelSeedProducerForElectrons_cff.py

```diff
@@ -19,9 +19,8 @@
     'FPix1_pos+FPix2_pos',
     'FPix1_neg+FPix2_neg',
 )
-globalPixelSeedsForElectrons.skipSimTrackIdTags = [cms.InputTag("globalPixelStepIds")]
-globalPixelSeedsForElectrons.outputSeedCollectionName = 'GlobalPixel'
-globalPixelSeedsForElectrons.zVertexConstraint = cms.double(0.5)
+globalPixelSeedsForElectrons.simTrackSelection.skipSimTrackIds = [cms.InputTag("globalPixelStepIds")]
+#globalPixelSeedsForElectrons.zVertexConstraint = cms.double(0.5)
 globalPixelSeedsForElectrons.originRadius = cms.double(0.02)
 globalPixelSeedsForElectrons.originpTMin = cms.double(1.5)
 
```

Comment:
@matt-komm @shchauha 

how did we arrive at the values for originRadius and originpTMin?
why is the zVertexConstraint removed?